server.go

```diff
@@ -899,7 +899,7 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 		ChainHash:              *s.cfg.ActiveNetParams.GenesisHash,
 		IncomingBroadcastDelta: lncfg.DefaultIncomingBroadcastDelta,
 		OutgoingBroadcastDelta: lncfg.DefaultOutgoingBroadcastDelta,
-		NewSweepAddr:           newSweepPkScriptGen(cc.Wallet),
+		NewSweepAddr:           newSweepPkScriptGen(cc.Wallet.WalletController),
```

Comment:
Why do all these need the underlying interface to be passed directly now?

Reply:
I think you can remove all diffs in this file and `subrpcserver_config.go` now with the changed method name.

Reply:
Correct, they've been removed.